common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,276 @@
+			this = culture:baradin
+			##Vrykul
+			this.culture_group = culture_group:vrykul_group
+			##Pirates
+			this = culture:pirate_culture
+			this = culture:blackwater
+			##Trolls
+			this.culture_group = culture_group:troll_group
+			##Ogres
+			this.culture_group = culture_group:ogre_group
+		}
+	}
+	modifier = {
+		naval_movement_speed_mult = 0.25
+		embarkation_cost_mult = -0.75
+		garrison_size = -0.30
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_lorewalking = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @lorewalking
+	potential = {
+		##Pandaren
+		this.culture_group = culture_group:pandaren_group
+	}
+	modifier = {
+		monthly_lifestyle_xp_gain_mult = 0.05
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_hive_breeding = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @hive_breeding
+	potential = {
+		##Aqir
+		this.culture_group = culture_group:aqir_group
+	}
+	modifier = {
+		levy_reinforcement_rate = 0.2
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_waterspeaking = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @waterspeaking
+	potential = {
+		##Jinyu
+		this.culture_group = culture_group:jinyu_group
+	}
+	modifier = {
+		naval_movement_speed_mult = 0.1
+		caster_damage_mult = 0.1
+		caster_toughness_mult = 0.1
+		caster_pursuit_mult = 0.05
+		caster_screen_mult = 0.05
+		knight_effectiveness_mult = 0.03
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+
+####################
+##Early Medieval##
+####################
+innovation_time_is_money = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @goblin
+	potential = {
+		##Goblin
+		this = culture:goblin	
+	}
+	modifier = {
+		monthly_income_mult = 0.05
+		vassal_tax_contribution_mult = 0.1
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_elite_ranks = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @elite_ranks
+	potential = {
+		##Human
+		this.culture_group = culture_group:human_group
+		NOT = {
+			this = culture:wastewander
+			this = culture:pirate_culture
+		}
+	}
+	modifier = {
+		knight_limit = 1
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_epidemic = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @epidemic
+	potential = {
+		##Undead
+		this.culture_group = culture_group:undead_group
+	}
+	custom = epidemic_conversion_bonus
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_architecture = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @zandalar
+	potential = {
+		##Zandalari
+		this = culture:zandalari
+	}
+	modifier = {
+		build_gold_cost = -0.5
```

Comment:
You made it OP AF. It must be the character modifier. Check other modifiers as well.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104380632-bd7aa280-5544-11eb-8070-51ede285d28d.png)

</details>